tests/plugins/webp-uploads/settings-tests.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Tests for webp-uploads plugin settings.php.
+ *
+ * @package webp-uploads
+ */
+
+class WebP_Uploads_Settings_Tests extends WP_UnitTestCase {
+
+	/**
+	 * @covers ::webp_uploads_add_settings_action_link
+	 */
+	public function test_webp_uploads_add_settings_action_link() {
+		$this->assertSame( 10, has_filter( 'plugin_action_links_' . WEBP_UPLOADS_MAIN_FILE, 'webp_uploads_add_settings_action_link' ) );
+		$this->assertFalse( webp_uploads_add_settings_action_link( false ) );
+
+		$default_action_links = array(
+			'deactivate' => '<a href="plugins.php?action=deactivate&amp;plugin=webp-uploads%2Fload.php&amp;plugin_status=all&amp;paged=1&amp;s&amp;_wpnonce=48f74bdd74" id="deactivate-speculation-rules" aria-label="Deactivate Speculative Loading">Deactivate</a>',
```

Comment:
Why `Speculative Loading` id and aria-label?

Reply:
I just copied it from what WP generates on core. The text doesn't matter, as long as it passes through unchanged.

Reply:
oh, I copied the Deactivate link from the wrong plugin

Reply:
Fixed in a6036989